### PR TITLE
make kafka topic rule clear for avro codec

### PR DESCRIPTION
**Pull request by zhangyangyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -301,7 +301,14 @@ URI 中可配置的的参数如下：
 {{< copyable "shell-regular" >}}
 
 ```shell
---sink-uri="kafka://127.0.0.1:9092/topic-name?&protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081"
+--sink-uri="kafka://127.0.0.1:9092/topic-name?&protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081" --config="changefeed_configs"
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:304]: --config="changefeed_configs" 这是啥？这么写挺奇怪的，虽然能用。
--config changefeed_config.toml 是不是这样好点

File: ticdc/ticdc-avro-protocol.md [hunk 2]
```diff
@@ -16,7 +16,14 @@ Avro 是由 [Apache Avro™](https://avro.apache.org/) 定义的一种数据交
 {{< copyable "shell-regular" >}}
 
 ```shell
-cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro" --sink-uri="kafka://127.0.0.1:9092/topic-name?kafka-version=2.6.0&protocol=avro" --schema-registry=http://127.0.0.1:8081
+cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro" --sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro" --schema-registry=http://127.0.0.1:8081 --config=changefeed_configs
```
Comment (leoppro) [inline ticdc/ticdc-avro-protocol.md:19]: 同上

File: ticdc/ticdc-avro-protocol.md [hunk 3]
```diff
@@ -34,9 +41,16 @@ cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro
 {{< copyable "shell-regular" >}}
 
 ```shell
-cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro-enable-extension" --sink-uri="kafka://127.0.0.1:9092/topic-name?kafka-version=2.6.0&protocol=avro&enable-tidb-extension=true" --schema-registry=http://127.0.0.1:8081
+cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro-enable-extension" --sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro&enable-tidb-extension=true" --schema-registry=http://127.0.0.1:8081 --config=changefeed_configs
```
Comment (leoppro) [inline ticdc/ticdc-avro-protocol.md:44]: 同上

File: ticdc/ticdc-avro-protocol.md [hunk 4]
```diff
@@ -186,14 +200,21 @@ Column 数据格式即 Key/Value 数据格式中的 `{{ColumnValueBlock}}` 部
 - `avro-bigint-unsigned-handling-mode` 决定了如何处理 BIGINT UNSIGNED 字段，它有两个选项：
 
     - string：Avro 将 BIGINT UNSIGNED 字段以 string 的方式处理。
-    - long：Avro 将 BIGINT UNSIGNED 字段以 64 位有符号整数处理，值可能会溢出。
+    - long：Avro 将 BIGINT UNSIGNED 字段以 64 位有符号整数处理，大于 9223372036854775807 的值会变为负数。
```
Comment (leoppro) [inline ticdc/ticdc-avro-protocol.md:203]:
```suggestion
    - long：Avro 将 BIGINT UNSIGNED 字段以 64 位有符号整数处理，大于 9223372036854775807 的值会发生溢出。
```

File: ticdc/ticdc-avro-protocol.md [hunk 5]
```diff
@@ -186,14 +200,21 @@ Column 数据格式即 Key/Value 数据格式中的 `{{ColumnValueBlock}}` 部
...
 
 配置样例如下所示：
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro-enable-extension" --sink-uri="kafka://127.0.0.1:9092/topic-name?kafka-version=2.6.0&protocol=avro&avro-decimal-handling-mode=string&avro-bigint-unsigned-handling-mode=string" --schema-registry=http://127.0.0.1:8081
+cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro-string-option" --sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro&avro-decimal-handling-mode=string&avro-bigint-unsigned-handling-mode=string" --schema-registry=http://127.0.0.1:8081 --config=changefeed_configs
```
Comment (leoppro) [inline ticdc/ticdc-avro-protocol.md:210]: 同上

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (zhangyangyu) [inline ticdc/manage-ticdc.md:304]:
```suggestion
--sink-uri="kafka://127.0.0.1:9092/topic-name?&protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081" --config changefeed_config.toml"
```

File: ticdc/ticdc-avro-protocol.md (hunk 2 above)
Comment (zhangyangyu) [inline ticdc/ticdc-avro-protocol.md:19]:
```suggestion
cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro" --sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro" --schema-registry=http://127.0.0.1:8081 --config changefeed_config.toml
```

File: ticdc/ticdc-avro-protocol.md (hunk 3 above)
Comment (zhangyangyu) [inline ticdc/ticdc-avro-protocol.md:44]:
```suggestion
cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro-enable-extension" --sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro&enable-tidb-extension=true" --schema-registry=http://127.0.0.1:8081 --config changefeed_config.toml
```

File: ticdc/ticdc-avro-protocol.md (hunk 5 above)
Comment (zhangyangyu) [inline ticdc/ticdc-avro-protocol.md:210]:
```suggestion
cdc cli changefeed create --pd=http://127.0.0.1:2379 --changefeed-id="kafka-avro-string-option" --sink-uri="kafka://127.0.0.1:9092/topic-name?protocol=avro&avro-decimal-handling-mode=string&avro-bigint-unsigned-handling-mode=string" --schema-registry=http://127.0.0.1:8081 --config changefeed_config.toml
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -301,7 +301,14 @@ URI 中可配置的的参数如下：
 {{< copyable "shell-regular" >}}
 
 ```shell
---sink-uri="kafka://127.0.0.1:9092/topic-name?&protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081"
+--sink-uri="kafka://127.0.0.1:9092/topic-name?&protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081" --config changefeed_config.toml"
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:304]:
```suggestion
--sink-uri="kafka://127.0.0.1:9092/topic-name?&protocol=avro&replication-factor=3" --schema-registry="http://127.0.0.1:8081" --config changefeed_config.toml
```

**Review by leoppro — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**